src/util/format.ts

```diff
@@ -227,7 +234,7 @@ export function getTooltipMarker(inOpt: ColorString | GetTooltipMarkerOpt, extra
             // Only support string
             + encodeHTML(color) + ';' + (extraCssText || '') + '"></span>'
         : '<span style="display:inline-block;margin-right:4px;'
-            + 'border-radius:10px;width:10px;height:10px;background-color:'
+            + 'border-radius:' + size + ';width:' + size + ';height:' + size + ';background-color:'
```

Comment:
you may want to use ` symbol so that you can use ${size} instead of '+ size + '. 1 character less to type.